src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.scss

```diff
@@ -7,5 +7,10 @@
 }
 
 .color-neutral {
-  color: gray;
+  color: green;
+  font-weight: bold;
+}
+
+.color-black {
+  color: black;
 }
```

Comment:
Clarification: Could you help check if the default is already black? We might not need another class for this.

Reply:
The default is not black.

Reply:
What is the default color? If it's the default dark gray I would be keen on keeping that instead. Pure black is usually not recommended for text on white backgrounds, usually only used for accessibility reasons (high contrast themes).

Reply:
I didn't want grey because it appears like disabled. Perhaps not pure black but a 'lighter' black? This is what Microsoft Forms uses
![image](https://user-images.githubusercontent.com/1673303/176115744-2179d06b-3160-44f2-a9bf-5ec716659db6.png)